src/SourceLocator/Type/AutoloadSourceLocator.php

```diff
@@ -163,18 +174,34 @@ private function locateClassByName(string $className) : ?string
 
         self::$autoloadLocatedFile = null;
         self::$currentAstLocator   = $this->astLocator; // passing the locator on to the implicitly instantiated `self`
-        $previousErrorHandler      = set_error_handler(static function (int $errno, string $errstr) : bool {
-            return true;
-        });
-        stream_wrapper_unregister('file');
-        stream_wrapper_register('file', self::class);
-        class_exists($className);
-        stream_wrapper_restore('file');
-        set_error_handler($previousErrorHandler);
+
+        $this->setErrorHandler();
```

Comment:
The internal function is `set_error_handler()`, but what we do on our end is silencing the error handler.

We should therefore call this `->silenceErrors()` or such

Reply:
Renamed.